src/pug/pages/footer.pug

```diff
@@ -1,11 +1,10 @@
 include ../constants/footer-category.pug
 footer.footer
+  img.footer__element-bottom(
+    src='./assets/images/element/element-3.webp',
```

Comment:
WebP didn't works on old browser (Safari), check it here: https://caniuse.com/webp
So, we might need 2 image type, webp as default and other format as fallback